tests/python_package_test/test_engine.py

```diff
@@ -18,7 +18,7 @@
 
 import lightgbm as lgb
 
-from .utils import (load_boston, load_breast_cancer, load_digits, load_iris, make_synthetic_regression,
+from utils import (load_boston, load_breast_cancer, load_digits, load_iris, make_synthetic_regression,
```

Comment:
@shiyu1994 Sure! Thanks a lot for adding the test!

Let's fix imports first.

```suggestion
from .utils import (load_boston, load_breast_cancer, load_digits, load_iris, make_synthetic_regression,
```